README.md

```diff
@@ -47,50 +42,50 @@ Ensure you have `kubectl` and `kustomize` installed in your `PATH`.
 Clone this repository and move to the repo folder:
 
 ```
-make deploy
-# /home/prometherion/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-# cd config/manager && /usr/local/bin/kustomize edit set image controller=quay.io/clastix/capsule:latest
-# /usr/local/bin/kustomize build config/default | kubectl apply -f -
-# namespace/capsule-system created
-# customresourcedefinition.apiextensions.k8s.io/tenants.capsule.clastix.io created
-# clusterrole.rbac.authorization.k8s.io/capsule-proxy-role created
-# clusterrole.rbac.authorization.k8s.io/capsule-metrics-reader created
-# clusterrolebinding.rbac.authorization.k8s.io/capsule-manager-rolebinding created
-# clusterrolebinding.rbac.authorization.k8s.io/capsule-proxy-rolebinding created
-# secret/capsule-ca created
-# secret/capsule-tls created
-# service/capsule-controller-manager-metrics-service created
-# service/capsule-webhook-service created
-# deployment.apps/capsule-controller-manager created
-# mutatingwebhookconfiguration.admissionregistration.k8s.io/capsule-mutating-webhook-configuration created
-# validatingwebhookconfiguration.admissionregistration.k8s.io/capsule-validating-webhook-configuration created
+$ git clone https://github.com/clastix/capsule
+$ cd capsule
+$ make deploy
 ```
 
-Log verbosity of the Capsule controller can be increased by passing the `--zap-log-level` option with a value from `1` to `10` or the [basic keywords](https://godoc.org/go.uber.org/zap/zapcore#Level) although it is suggested to use the `--zap-devel` flag to get also stack traces.
+It will install the Capsule controller in a dedicated namespace `capsule-system`.
 
-During startup Capsule controller will create additional ClusterRoles `capsule-namespace-deleter`, `capsule-namespace-provisioner` and ClusterRoleBinding `capsule-namespace-provisioner`. These resources are used in order to allow Capsule users to manage their namespaces in tenants.
+## Admission Controllers
+Capsule implements Kubernetes multi-tenancy capabilities using a minimum set of standard [Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) enabled on the Kubernetes APIs server. See the corresponding [section](documentation.md) in the documentation for a detailed list of required Admission Controllers.
```

Comment:
Same here: `documentation.md` doesn't exist as file.

Reply:
This PR is still a draft